doc/uftrace-recv.md

```diff
@@ -27,6 +27,36 @@ Data will be sent using `uftrace-record` with -H/\--host option.
 :   Run given (shell) command as soon as receive data.  For example, one can run "uftrace replay" for received data.
 
 
+EXAMPLE
+=======
+ The uftrace recv command should be run before sending data by record command.
+
+    # HOST 
+    $ uftrace recv -d recv_data --port 1234
+
+ Above command starts a server with port by given (default `8090`.) to receive data from remote client. 
+
+    # CLIENT :
+    $ uftrace record -H localhost -d example_data --port 1234 example
+
+ Above command sends the trace data to a remote server that pointed by given -H option (`localhost` in this case) after running the example program.  and as you see, you can choose save directory by using `-d` option (sending data will save to `example_data` in this case).
```

Comment:
I'd like to break long lines to fit in 80-column (like source code).  And please use an upper case letter for and --> "And as you see".